### PR TITLE
INS-79: Add root record on DB init

### PR DESCRIPTION
**Pull request by andreyromancev**
closes INS-79

**- What I did**
Implemented initial storage record creation

**- How to verify it**
Run tests

### CODE REVIEW

File: ledger/storage/leveldb/leveldb.go [hunk 1]
```diff
@@ -96,9 +99,28 @@ func InitDB() (*LevelLedger, error) {
...
+	ledger := LevelLedger{
 		ldb: db,
-	}, nil
+		zeroRef: record.Reference{
+			Domain: record.ID{}, // TODO: fill domain
+			Record: zeroID,
+		},
+	}
+	_, err = db.Get([]byte(zeroRecordHash), nil)
+	switch err {
```
Comment (nordicdyno) [inline ledger/storage/leveldb/leveldb.go]: @andreyromancev @ita-sammann Error checking in this way is not idiomatic go code. I would like to see this part rewritten

File: Gopkg.lock [hunk 2]
```diff
@@ -102,31 +153,54 @@
...
+    "github.com/satori/go.uuid",
+    "github.com/spf13/pflag",
+    "github.com/stretchr/testify/assert",
+    "github.com/stretchr/testify/mock",
+    "github.com/syndtr/goleveldb/leveldb",
+    "github.com/syndtr/goleveldb/leveldb/comparer",
+    "github.com/syndtr/goleveldb/leveldb/opt",
+    "github.com/ugorji/go/codec",
+    "golang.org/x/crypto/sha3",
+  ]
```
Comment (nordicdyno) [inline Gopkg.lock]: @andreyromancev @ita-sammann why so different format? Probably we should fix dep version for consistent packages versioning.

Comment (ita-sammann) [reply Gopkg.lock]: @nordicdyno, it's not clear for me. These changes are no more in master and we can't reproduce this behaviour of `dep ensure`.

Comment (ita-sammann) [reply Gopkg.lock]: Current dep version is 0.5.0 let's stick to it.

File: ledger/storage/leveldb/leveldb.go (hunk 1 above)
Comment (ita-sammann) [reply ledger/storage/leveldb/leveldb.go]: It makes sense. Let's use if/else if/else in similar cases.

**Pull request merged**